recipes/dmd/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+
+cd $SRC_DIR/linux/bin64/
+
+chmod a+x ddemangle dman dmd dumpobj dustmite obj2asm rdmd
+
+cp ddemangle dman dmd dumpobj dustmite obj2asm rdmd $PREFIX/bin
```

Comment:
I believe this is the standard way provided by D lang to install their compilers. I have no experience with the language neither its compilation. I am pulling this recipe to solve a dependency to another tool I am currently uploading to the bioconda channel.